src/Serilog.Sinks.XUnit/Serilog.Sinks.XUnit.csproj

```diff
@@ -26,11 +26,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="Serilog" Version="2.12.0" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
```

Comment:
FYI - Source Link is now included in .NET SDK 8 and enabled by default see https://github.com/dotnet/sourcelink/releases

Reply:
Want me to remove it?